ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -27,11 +28,11 @@ public struct ShadowInfo: Equatable {
                 xTwo: CGFloat,
                 yTwo: CGFloat) {
         self.colorOne = colorOne
-        self.blurOne = blurOne
+        self.blurOne = blurOne * shadowBlurAdjustment
```

Comment:
@lyzhan7 : so for FURN global token is iOS shawdow already halved from the token definition? or is furn programmatically dividing it by 2?

Reply:
FURN programmatically divides by 2. https://github.com/microsoft/fluentui-react-native/blob/3f16965e77fc0e4aedd093b2206a2e523c6bcdca/packages/experimental/Shadow/src/shadowStyle.ts#L11

Reply:
Sorry my FUA emails are going to a different folder, still trying to figure out the right email filters. Yes we decided that we should programmatically divide by 2